examples/data-objects/clicker/src/index.tsx

```diff
@@ -45,26 +55,19 @@ export class Clicker extends DataObject implements IFluidHTMLView {
         const taskManagerHandle = this.root.get<IFluidHandle<TaskManager>>(taskManagerKey);
         this._taskManager = await taskManagerHandle?.get();
 
+        this.counter.on("incremented", () => { this.emit("incremented"); });
         this.setupAgent();
     }
 
-    // #region IFluidHTMLView
-
-    /**
-     * Will return a new Clicker view
-     */
-    public render(div: HTMLElement) {
-        // Get our counter object that we set in initialize and pass it in to the view.
-        ReactDOM.render(
-            <CounterReactView counter={this.counter} />,
-            div,
-        );
-        return div;
+    public increment() {
```

Comment:
nit: would we want to open up the API to take in an optional value parameter to increment? In case somebody wanted to increment by more than 1

Reply:
We could - the current sample only does increments of 1, so I left it as-is.  Since it's more of a illustrative example than a "useful" component, I'm not particularly worried about it.